src/helpers/types.ts

```diff
@@ -117,6 +117,8 @@ export type DatatableProps = {
   labels?: LabelType;
   /** Custom table components. */
   Components?: TableComponents;
+  /** Show/hide pagination even if there is only 1 page */
```

Comment:
Very minor nit: always end with a period for comments if the sentence is finished:

```suggestion
  /** Show/hide pagination even if there is only 1 page. */
```